schemas/event.yml

```diff
@@ -68,19 +69,31 @@
             It is used to represent events that indicate that something happened.
         - name: metric
           description: >
-            This value is used to indicate that this event that a numeric measurement
-            was taken at given point in time.
+            This value is used to indicate that this event describes a numeric measurement
+            taken at given point in time.
 
-            Examples include CPU utilization, memory usage, or a vulnerability scan result.
+            Examples include CPU utilization, memory usage, or device temperature.
 
             Metric events are often collected on a predictable frequency, such as once
             every few seconds, or once a minute.
         - name: state
           description: >
-            This value is similar to metric, except that the entity being measured does not
-            provide a numeric metric value, but rather one of a fixed set of conditions or states.
-            For example a periodic event reporting a "fin_wait" state of a TCP connection
-            on a host might use `event.type:state`.
+            The state value is similar to metric, indicating that this event describes a
+            measurement taken at given point in time, except that the measurement does not
+            result in a numeric value, but rather one of a fixed set of categorical values
+            that represent conditions or states.
+
+            Examples include periodic events reporting Elasticsearch cluster state (green/yellow/red),
+            the state of a TCP connection (open, closed, fin_wait, etc.), the state of a host with respect
+            to a software vulnerability (vulnerable, not vulnerable), and the state of a system
+            regarding compliance with a regulatory standard (compliant, not compliant).
+
+            Note that an event that describes a change of state would not use `event.kind:state`,
+            but instead would use 'event.kind:event' since a state change fits the more general
+            event definition of something that happened.
+
+            State events are often collected on a predictable frequency, such as once
+            every few seconds, once a minute, once an hour, or once a day.
```

Comment:
will "often collected on a predictable frequency" suggest to casual users that a one off, or manually initiated ad hoc state query (vulnerability scan for instance) should go somewhere else?  
should it say something like:
            State events are often collected on a predictable frequency, such as once
            every few seconds, once a minute, once an hour, or once a day, **but can 
            also be used to store ad-hoc state queries as performed**

Reply:
We added "often" for exactly that reason - to allow for ad hoc or manual measurements to still be included in this event kind, but because we are writing a spec, I like your suggestion for explicit clarity.  Will add.